erpc_c/port/erpc_setup_extensions_freertos.cpp

```diff
@@ -16,6 +16,9 @@ using namespace erpc;
 
 static Semaphore *s_erpc_call_in_progress = NULL;
 static TimerHandle_t s_erpc_call_timer_cb = NULL;
+#if configSUPPORT_STATIC_ALLOCATION
```

Comment:
Condition in contradiction with #169 , should be changed to the way used in #169, i.e.
#if ERPC_ALLOCATION_POLICY == ERPC_ALLOCATION_POLICY_STATIC

applicable on more places